src/ooxml/java/org/apache/poi/xslf/usermodel/XMLSlideShow.java

```diff
@@ -416,7 +425,7 @@ public XSLFNotesMaster getNotesMaster() {
      * Return all the charts in the slideshow
      */
     public List<XSLFChart> getCharts() {
-        return _charts;
+        return Collections.unmodifiableList(_charts);
```

Comment:
changed it to unmodifiable list as we have unmodifiable list in XWPFDocument and XSSFWorkbook.